docs/conf.py

```diff
@@ -78,6 +78,8 @@
     "https://crates.io/",
     # Ignore releases, since we'll include the next release before it exists.
     "https://github.com/canonical/[a-z]*craft[a-z-]*/releases/.*",
+    # returns a 403 from GitHub CI
+    "https://rsync.samba.org",
```

Comment:
I assumed this was a temporary issue but it's been broken for 4-5 weeks.  

It's a URL that's unlikely to change so I'm not worried about ignoring it.